flanker/addresslib/address.py

```diff
@@ -56,7 +56,7 @@
 
 
 @metrics_wrapper()
-def parse(address, addr_spec_only=False, metrics=False):
+def parse(address, addr_spec_only=False, metrics=False, fallback_last_word=False):
```

Comment:
`fallback_last_word` should be `True` by default. This way we more or less preserve original behaviour but also provide a way to be strict if we want. And I would call this parameter `strict=False`, neither naming makes the parameter semantic immediately obvious so people would need to read the docstring anyway.